packages/legacy-events/ReactGenericBatching.js

```diff
@@ -77,12 +75,12 @@ export function batchedEventUpdates(fn, a, b) {
   }
 }
 
-export function executeUserEventHandler(fn: any => void, value: any) {
+// This is for the React Flare event system
+export function executeUserEventHandler(fn: any => void, value: any): any {
   const previouslyInEventHandler = isInsideEventHandler;
   try {
     isInsideEventHandler = true;
-    const type = typeof value === 'object' && value !== null ? value.type : '';
-    invokeGuardedCallbackAndCatchFirstError(type, fn, undefined, value);
```

Comment:
Is there a follow-up to fix this? It seems important that event handler doesn't crash the rest of the code?

Reply:
Yeah, I plan on addressing it this week.

Reply:
This was already fixed in a follow-up right? This guard is still there on master

Reply:
LOL. I did, I’m clearly losing the plot.